lightly/transforms/rotation.py

```diff
@@ -50,10 +50,30 @@ def __call__(self, sample):
         return sample
 
 class RandomRotateDegrees(object):
+    """Random rotate image between two rotation angles with a random probability.
+
+    Attributes:
+        prob:
+            Probability with which image is rotated.
+        degrees (sequence or number): Range of degrees to select from.
+            If degrees is a number instead of sequence like (min, max), the range of degrees
+            will be (-degrees, +degrees).
```

Comment:
Also please add a description how the rotation works. -> We select a random angle in (min, max) or (-degrees, +degrees) and we rotate counter-clockwise.